python/pyspark/sql/functions.py

```diff
@@ -29,9 +29,9 @@
 from pyspark.sql.dataframe import DataFrame
 from pyspark.sql.types import StringType, DataType
 # Keep UserDefinedFunction import for backwards compatible import; moved in SPARK-22409
-from pyspark.sql.udf import UserDefinedFunction, _create_udf
```

Comment:
`UserDefinedFunction` should be kept as described above.

Reply:
Thanks!